django/utils/http.py

```diff
@@ -293,12 +295,21 @@ def is_safe_url(url, host=None):
             url = force_text(url)
         except UnicodeDecodeError:
             return False
+    if allowed_hosts is None:
+        allowed_hosts = set()
+    if host:
+        warnings.warn(
```

Comment:
Do we really need to deprecate this? I did a quick Github search and it seems to me that there are a lot of projects using `is_safe_url()` with a single `host` argument, named and unnamed.